pkg/diagnostics/collectors.go

```diff
@@ -402,8 +407,8 @@ func (c *collectorFactory) hostPortCollector(ports []string, hostIP string) *Col
 	argsIP := []string{tempIPRequest}
 	return &Collect{
 		RunPod: &runPod{
-			Name:       "run-ip",
-			Namespaces: constants.EksaDiagnosticsNamespace,
+			Name:      "run-ip",
```

Comment:
we should probably rename these to something more clear instead of just `run-ip`

Reply:
How about `check-host-port` and next could be `ping-host-ip`? Any suggestions?